utils/scripts/ocm/ocm.py

```diff
@@ -844,6 +848,73 @@ def update_notification_email_address(self, addon_name, email_address, exit_on_f
             else:
                 return False
 
+    def get_latest_osd_candidate_version(self, osd_major_version, osd_minor_version):
+        """
+        get the latest  candidate version
+        Args:
+            osd_major_version (str):  Major version of the release. For example in 4.8.10 : 4 is major version
+            osd_minor_version (str):  Major version of the release. For example in 4.8.10 : 4 is minor version
```

Comment:
Change the description of minor version

Reply:
ack

Reply:
is it solved? I still see the old version

Reply:
I keep seeing the old version, though
![image](https://user-images.githubusercontent.com/88311595/172355072-cc5bd569-e18e-4011-93bc-246d2445d499.png)